components/ExerciseCard/ExerciseCard.tsx

```diff
@@ -10,15 +10,24 @@ export type ExerciseCardProps = {
   explanation: string
 }
 
+const messages = {
```

Comment:
Why an object instead of an `enum`?

Reply:
There's no specific reason, so I'll switch this to an enum. 
```typescript
const messages = {
  EMPTY: '',
  ERROR: 'Your answer is incorrect - please try again.',
  SUCCESS: '🎉 Your answer is correct!'
} as const
type MessageKey = keyof typeof messages
```
```typescript
enum Message {
  EMPTY = '',
  ERROR = 'Your answer is incorrect - please try again.',
  SUCCESS = '🎉 Your answer is correct!'
}
type MessageKey = keyof typeof Message
```